### ISSUE TITLE
Summaries: Include arbitrary tags in templates

### ISSUE THREAD

**Issue by dwertheimer** [NotePlan/plugins#362]
Progress updates are limited to whatever I list in the preferences, but I want to be able to specify specific summaries in my templates and ignore/override the preferences.
Something like (see the last param below):

```
<%- progressUpdate({period: '2022-11-04', heading: 'Habits', showSparklines: true, includeToday:false, yesNoItems:"#tennis"}) %>
```

I would suggest that if anyone sets any type of item (yesNoItems or hashtagCounts or whatever), it overrides all the other types and makes them null. Because I think it's pretty clear from my tag above that I only want the #tennis checkmarks and not the rest of the stuff in the prefs.

Seems like it would be fairly straightforward to do so because you're already doing some config overrides by the time you get to this call in your code (pasted here just for reference): 

```js
    const tmOccurrencesArray = await gatherOccurrences(
      periodString,
      fromDateStr,
      toDateStr,
      config.progressHashtags,
      [],
      config.progressMentions,
      [],
      config.progressYesNo,
      config.progressMentions,
      config.progressMentionsAverage,
      config.progressMentionsTotal,
    )
    ```

**Comment by jgclark** [comment]
Yes, should be possible.
But, boy, the README is getting more and more complex!

Incidentally, this code is an example of something quite simple growing, and growing, and ... becoming quite ungainly. Possibly needing some refactoring.

**Comment by jgclark** [comment]
Included in v0.16
